source/reference/limitations.txt

```diff
@@ -32,8 +32,16 @@ General Limitations
   6.2. Only major MongoDB releases such as 6.0 or 7.0. are supported.
   For more information on MongoDB versioning, see
   :ref:`release-version-numbers`.
-- The source and destination clusters must have the same release
-  version.
+- The source and destination clusters must have the same major and minor
```

Comment:
Could we update the first bullet in this section to also say that the min supported server versions are 6.0.8 and 7.0.0?